gapis/api/vulkan/footprint_builder.go

```diff
@@ -1992,6 +1992,9 @@ func (vb *FootprintBuilder) BuildFootprint(ctx context.Context,
 			ft.AddBehavior(ctx, cbh)
 		}
 
+		bh.Alive = true // TODO(awoloszyn): Investigate why this is needed.
```

Comment:
Probably a good idea to put the issue number here.
I could write a tool quite easily that scans our codebase for issues that have since been fixed.

Reply:
Done.